ideascube/serveradmin/tests/test_csv2pkg_command.py

```diff
@@ -41,6 +41,11 @@ def test_csv2pkg_command(package_path, csv_writer, capsys):
                            }
 
     assert "Package {} has been created with 3 medias.".format(package_path) in out
+    #Check the order of the output
+    for k in ('name', 'description', 'language', 'version', 'url', 'size', 'sha256sum', 'type'):
+        index = out.find('    '+k)
+        assert index != -1, '{} is not in the right order'.format(k)
+        out = out[index:]
```

Comment:
As `out` is not used elsewhere it's more a matter of style.
But I don't especially care, I will change it :)